### ISSUE TITLE
Add .editorconfig file to project scaffolded by Rekit.

### ISSUE THREAD

**Issue by vunguyenhung** [supnate/rekit#85]
Firstly, thank you for an amazing product/development experience.

I think it would be nice if we have a `.editorconfig` file at the project root to support people wanted to use different IDE (e.g., I'm using Rekit as supporting tool and WebStorm as IDE)?

My suggestion for the content of that file (it's Airbnb's `.editorconfig`):
```
root = true

[*]
indent_style = space
indent_size = 2
charset = utf-8
trim_trailing_whitespace = true
insert_final_newline = true
end_of_line = lf

```

Thanks.

**Comment by supnate** [comment]
You can add it by yourself. Rekit project is just a normal React project you can use any editors on it.

**Comment by vunguyenhung** [comment]
Yes, I actually manually added it myself. I just suggest adding it automatically during scaffolding process.
